src/domains/ecommerce-common/00_alerts.tf

```diff
@@ -134,3 +134,37 @@ AzureDiagnostics
     threshold = 60 // NDP for each requests does 5 retries with a 2 minutes interval after a 10 minutes delay
   }
 }
+
+
+# eCommerce NPG monitoring: KO or slow payment methods start session api call (order/build retrieve card form fields)
+resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_payment_methods_start_session_alert" {
+  count = var.env_short == "p" ? 1 : 0
+
+  name                = "ecommerce-payment-methods-start-session-alert"
+  resource_group_name = azurerm_resource_group.rg_ecommerce_alerts[0].name
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id, azurerm_monitor_action_group.ecommerce_opsgenie[0].id]
```

Comment:
We should add alerts on Opsgenie only when we can consider  tuned enough.

Reply:
fixed with https://github.com/pagopa/pagopa-infra/pull/1956/commits/fb0a8d17193737295f100d5c7b0c33bf396b5922